data/data-pipeline/data_pipeline/etl/score/etl_score_geo.py

```diff
@@ -155,24 +161,24 @@ def _aggregate_to_tracts(
         self, block_group_df: gpd.GeoDataFrame
     ) -> gpd.GeoDataFrame:
         # The tract identifier is the first 11 digits of the GEOID
-        block_group_df["tract"] = block_group_df.apply(
-            lambda row: row[self.GEOID_FIELD_NAME][0:11], axis=1
-        )
+        block_group_df["tract"] = block_group_df[self.GEOID_FIELD_NAME].str[:11]
         state_tracts = block_group_df.dissolve(by="tract", aggfunc="mean")
         return state_tracts
 
     def _create_buckets_from_tracts(
         self, state_tracts: gpd.GeoDataFrame, num_buckets: int
     ) -> gpd.GeoDataFrame:
-        # assign tracts to buckets by D_SCORE
+        # assign tracts to buckets by score
```

Comment:
here's where i think we need to do some thinking. 

right now, WY has score buckets that get absorbed into 0. This is where the buckets are assigned. It might make sense to ensure that all buckets are homogeneous, so either all 0 or all 1.

Reply:
So... I think this initial thought was wrong. I did implement code to make the buckets more homogeneous, but that is not what was going on with WY.

Reply:
(There COULD be something going on with the compression in `_breakup_multipolygons` but I'm not sure?)